scripts/init.sh

```diff
@@ -6,4 +6,10 @@ PROJECT_NAME="$1"
 HOST='' \
 PORT='' \
 PROJECT='' \
-docker-compose run server python mage_ai/cli/main.py init $PROJECT_NAME
+docker_version=$(docker version --format '{{.Server.Version}}')
+if [[ "$docker_version" == 2*.* ]]; then
+  docker compose run server python mage_ai/cli/main.py init $PROJECT_NAME
```

Comment:
Could you move the HOST, PORT, PROJECT to be inside the if else branch as well?
``
HOST='' \
PORT='' \
PROJECT='' \
docker compose run server python mage_ai/cli/main.py init $PROJECT_NAME
```

Reply:
@wangxiaoyou1993 updated init.sh file